wrappers/python/nntile/layer/batch_norm.py

```diff
@@ -12,6 +12,7 @@
 # @version 1.0.0
 
 import math
+from typing import Self
```

Comment:
Sorry for that. It's a little bit tricky with `Self`. I just wanted to show proper type hinting but created another issue. :disappointed: 

```python
import sys

if sys.version_info < (3, 11):
    from typing_extensions import Self
else:
    from typing import Self
```

Minimum Python version is 3.10 in order to use `nntile` in Colab.

Reply:
removed